src/benchmark/tpcds_benchmark.cpp

```diff
@@ -35,11 +35,11 @@ int main(int argc, char* argv[]) {
 
   // clang-format off
   cli_options.add_options()
-    ("s,scale", "Database scale factor (1 ~ 1GB)", cxxopts::value<int>()->default_value("1"));
+    ("s,scale", "Database scale factor (1 ~ 1GB)", cxxopts::value<int32_t>()->default_value("1"));
```

Comment:
Could you explain why you changed this to int32_t? `int` means "I just want a reasonably sized integer", `int32_t` says that you actually thought about the width of the int.

Reply:
Hyrise's [contribution guidelines](https://github.com/hyrise/hyrise/blob/master/CONTRIBUTING.md#c-guidelines) led me to this change: `Use [u]int(8|16|32|64)_t instead of int, long, uint etc.`

Reply:
Whoopsie. We might want to revisit that.

Reply:
No need to change it here